### ISSUE TITLE
households_flexibility_p2p_electricity (Battery) should take conversion losses into account

### ISSUE THREAD

**Issue by grdw** [quintel/etmoses#786]
Input capacity is now equal to output capacity in ETMoses.

**Comment by ChaelKruip** [comment]
I am not sure how this would work in ETMoses though. P2P should actually have losses when charging and when discharging (as in ETE). I can imagine the P2P class needs some work to take this into account right @antw ?

**Comment by antw** [comment]
> I can imagine the P2P class needs some work to take this into account right @antw ?

Yup. Moses' support for efficiency isn't great, and since P2P and EV do weird things – such as emitting *potential* production for use by other techs, and then reclaiming unused energy back – it needs a bit of thought as to how to implement this cleanly. Same goes for the other issues regarding loss (#791, #799).

This seems quite substantial, so I think maybe we should pick this up after heat networks.
